.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Delete this gitignore file. It is redundant

Reply:
Done